src/full/Agda/Interaction/JSONTop.hs

```diff
@@ -287,17 +287,17 @@ instance EncodeTCM Blocker where
 instance EncodeTCM DisplayInfo where
   encodeTCM (Info_CompilationOk backend wes) = kind "CompilationOk"
     [ "backend"           @= encodePretty backend
-    , "warnings"          #= encodeTCM (filterTCWarnings (tcWarnings wes))
-    , "errors"            #= encodeTCM (filterTCWarnings (nonFatalErrors wes))
+    , "warnings"          #= (encodeTCM $ filterTCWarnings (tcWarnings wes))
+    , "errors"            #= (encodeTCM $ filterTCWarnings (nonFatalErrors wes))
```

Comment:
Haha, you are rewriting in the opposite direction of `hlint`.  This might clash with @philderbeast 's efforts in
- https://github.com/agda/agda/pull/6461

This is an observation only, I am not taking sides here.  Personally, I am not convinced that `hlint` is a productivity booster for this repo.